backend/src/v2/compiler/argocompiler/argo.go

```diff
@@ -83,7 +83,21 @@ func Compile(jobArg *pipelinespec.PipelineJob, kubernetesSpecArg *pipelinespec.S
 			}
 		}
 	}
-
+	// fill nested pipeline default parameters to PipelineJob
+	for _, componentSpec := range spec.Components {
+		if componentSpec.GetDag() != nil && componentSpec.GetInputDefinitions() != nil {
+			for param, paramSpec := range componentSpec.GetInputDefinitions().GetParameters() {
+				_, ok = job.RuntimeConfig.ParameterValues[param]
```

Comment:
Am I reading this correctly that it assumes the pipeline input parameter and the nested pipeline input parameter are the same name?